src/kontrol/prove.py

```diff
@@ -90,6 +90,10 @@ def foundry_prove(
         for test in test_suite:
             if not isinstance(test.method, Contract.Method) or test.method.function_calls is None:
                 continue
+            if not test.contract.has_storage_layout:
+                raise RuntimeError(
+                    "Couldn't locate 'storageLayout' in the compiled SOLC output. Please add `extra_output = ['storageLayout']` to your foundry.toml file."
```

Comment:
Obviously a nit, feel free to ignore pls
```suggestion
                    "Couldn't locate 'storageLayout' in the compiled solc output. Please add `extra_output = ['storageLayout']` to your foundry.toml file."
```

Reply:
Thanks!